lib/streamlit/elements/plotly_chart.py

```diff
@@ -165,14 +165,17 @@ def plotly_chart(
         # NOTE: "figure_or_data" is the name used in Plotly's .plot() method
         # for their main parameter. I don't like the name, but it's best to
         # keep it in sync with what Plotly calls it.
+        widget_callback = None
```

Comment:
nit: maybe better to call this: `on_select_callback` to use the same name as in altair